pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -250,10 +250,6 @@ const scheduleBackupOp = "CREATE SCHEDULE FOR BACKUP"
 func doCreateBackupSchedules(
 	ctx context.Context, p sql.PlanHookState, eval *scheduledBackupEval, resultsCh chan<- tree.Datums,
 ) error {
-	if err := p.RequireAdminRole(ctx, scheduleBackupOp); err != nil {
```

Comment:
does this actually check owner somewhere?

Reply:
Owner of? This is the `CREATE SCHEDULE` code path so theres no pre-existing schedule to check against. Unless I'm misunderstanding your question.

Reply:
Nope, I derped :) thanks!